docs/source/peer-chaincode-devmode.rst

```diff
@@ -0,0 +1,129 @@
+Using dev mode
+==============
+
+Normally chaincodes are started and maintained by peer. However in “dev”
+mode, chaincode is built and started by the user. This mode is useful
+during chaincode development phase for rapid code/build/run/debug cycle
+turnaround.
+
+To keep this a realistic “dev” environment, we are going to keep it “out
+of the box”.
+
+- Note: Make sure peer is not using TLS when running in dev mode.
+
+All commands are executed from the ``fabric`` folder.
+
+Setup environment
+-----------------
+Build binaries for orderer, peer and configtxgen.
+::
+    make orderer peer configtxgen
+::
+
+Setup PATH environment variable to include orderer and peer binaries.
+::
+    cd ./build/bin/; export PATH=$PATH:$(pwd)
```

Comment:
```suggestion
   export PATH=$(pwd)/build/bin:$PATH
```

This should be reversed, otherwise existing binaries on the path would take precedent. Also no reason to do the cd, just append it, makes subsequent commands clearer

Reply:
done.